pagopa/pn-frontend
# fix(PN-9968): update session history removing replace() method

## Short description
Update session history flow removing replace() method and using open() method with _self property in PF and PG

## How to test
Start PG/PF, click on "Accedi" button. When the page with SPID or CIE login is loaded, go back and check to be again in the page with only one "Accedi" button.
